message/message.go

```diff
@@ -131,22 +132,39 @@ func (m *Message) AddArg(arg string) *Message {
 	return m
 }
 
-// AsSlice outputs the given Message as a string slice.
-// The slice contains the string form of the Message's word in index 0, and
-// the arguments as index 1 upwards, if any.
-func (m *Message) AsSlice() []string {
-	slice := []string{m.word.String()}
-	for _, arg := range m.args {
-		slice = append(slice, arg)
-	}
-	return slice
```

Comment:
This is probably for the best, as it duplicates `Args`, but did you check to make sure nothing downstream is broken by losing this?